src/Pages/Signup/Signup.js

```diff
@@ -1,12 +1,278 @@
+                <td>
+                  <input
+                    className="inputbox-commonstyle"
+                    type="text"
+                    placeholder="숫자만 입력해주세요"
+                  />
+                  <button className="button gray-btn">인증번호 받기</button>
+                </td>
+              </tr>
+              <tr>
+                <th>
+                  주소<i className="sub">*</i>
+                </th>
+                <td>
+                  <button className="big-btn">
+                    <i class="fas fa-search"></i>주소검색
+                  </button>
+                  <p className="guide">
+                    <span>배송지에 따라 상품 정보가 달라질 수 있습니다.</span>
+                  </p>
+                </td>
+              </tr>
+              <tr className="sex">
+                <th>성별</th>
+                <td className="fw-400">
+                  <input type="radio" id="man" name="gender" />
+                  <label htmlFor="man">남자</label>
+
+                  <input type="radio" id="woman" name="gender" />
+                  <label htmlFor="woman">여자</label>
+
+                  <input type="radio" id="none" name="gender" />
+                  <label htmlFor="none">선택안함</label>
+                </td>
+              </tr>
+              <tr>
+                <th>생년월일</th>
+                <td>
+                  <div className="birth-input-wrap">
+                    <input type="text" placeholder="YYYY" maxlength="4" />
+                    <span>/</span>
+                    <input type="text" placeholder="MM" maxlength="2" />
+                    <span>/</span>
+                    <input type="text" placeholder="DD" maxlength="2" />
+                  </div>
+                  <p className="guide">
+                    <span className="red">x 태어난 월을 정확하게 입력해주세요.</span>
+                  </p>
+                </td>
+              </tr>
+              <tr className="sex">
+                <th>추가입력 사항</th>
+                <td className="fw-400">
+                  <input type="radio" id="id" name="recommend" />
+                  <label htmlFor="id">추천인 아이디</label>
+                  <input type="radio" id="event" name="recommend" />
+                  <label htmlFor="event">참여 이벤트명</label>
+                  <div className="joinevent">
+                    <input
+                      type="text"
+                      placeholder="참여 이벤트명을 입력해주세요"
+                      className="inputbox-commonstyle mt-24"
+                    />
+                    <div className="event-txt">
+                      <p className="guide">
+                        <span>추천인 아이디와 참여 이벤트면 중 하나만 선택 가능합니다.</span>
+                        <span>가입 이후, 수정이 불가능합니다.</span>
+                        <span>대소문자 및 띄어쓰기에 유의해주세요</span>
+                      </p>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            </table>
+            <div className="under-line"></div>
```

Comment:
content가 없는 요소는 아래와 같이 selfclosing 해주세요! jsx의 특징이죠! :) 
```suggestion
            <div className="under-line" />
```

Reply:
수정 했습니다 :)